.github/workflows/main.yml

```diff
@@ -37,8 +37,8 @@ jobs:
       ICEBERG_DIR: included-builds/iceberg
       ICEBERG_MAIN_REPOSITORY: apache/iceberg
       ICEBERG_MAIN_BRANCH: master
-      ICEBERG_PATCH_REPOSITORY: ''
-      ICEBERG_PATCH_BRANCH: ''
+      ICEBERG_PATCH_REPOSITORY: snazy/iceberg
```

Comment:
PITA to figure out that one of these Palantir plugins used in Iceberg causes issues